### DISCUSSION TITLE
Checkboxes not taking dark styles

### DISCUSSION THREAD

**Discussion by RichardHpa** [tailwindlabs/tailwindcss#9830] [Help]
**What version of Tailwind CSS are you using?**
3.2.4

**What version of Node.js are you using?**
v16.14.2

**What browser are you using?**
Chrome

**What operating system are you using?**
macOS

**Reproduction URL**
https://play.tailwindcss.com/WkCzbd6I3M

**Describe your issue**
I am trying to style a checkbox input using tailwind. I have taken some code from [flowbite](https://flowbite.com/docs/forms/checkbox/) which has the styles for light and dark mode all ready set. But neither of the light or dark mode styles for background-color work with inputs.

I have created a tailwind play to show you the example and have set the expected light mode background to be green and the dark mode to be red. I have set the mode to dark and the background color is just the default browers color. If I remove the dark class the input also just styles the same color.

I'm also not sure if there are any other elements that are being effected by this.

**Comment by thecrypticace** [comment]
Hey. the reason you're seeing a difference in behavior is `frostbite` is using our forms plugin which enables this behavior. If you install and add the plugin to your Tailwind CSS config you'll see the colors change as expected: https://play.tailwindcss.com/mK8JRyHl5E

**Reply by RichardHpa** [reply]
Ah well that makes sense. Though I do wonder why is the form plugin not just bundled into the main package rather than have it as its own package.

**Reply by adamwathan** [reply]
> Though I do wonder why is the form plugin not just bundled into the main package rather than have it as its own package.

Just so it can be versioned independently, and we don't have to release Tailwind v4 just to make a small breaking change to the forms plugin. We've never been fully satisfied with the forms plugin as a universal set of base styles (it's still too opinionated IMO to be in Tailwind core) which is why we've never been confident just including it in core.
